File: app/src/main/java/com/dynamo/spacex/services/SpaceXService.java
package com.dynamo.spacex.services;

import com.dynamo.spacex.models.PastLaunch;

import java.util.List;

import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface SpaceXService {

    @GET("launches/past")
    Single<List<PastLaunch>> getPastLaunch(@Query("limit") int limit);
}

File: app/src/main/java/com/dynamo/spacex/service/SpaceXService.kt
package com.dynamo.spacex.service

import com.dynamo.spacex.model.PastLaunch
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query

interface SpaceXService {

    @GET("launches/past")
    fun getPastLaunch(@Query("offset") offset:Int, @Query("limit") limit:Int): Single<List<PastLaunch>>
}